src/widget/tool/chattextedit.cpp

```diff
@@ -69,3 +70,54 @@ void ChatTextEdit::retranslateUi()
 {
     setPlaceholderText(tr("Type your message here..."));
 }
+
```

Comment:
it's not a QT problem, problem apparently is that the translation of these lines need a library that is not supplied with the qtox